src/components/ComputeModal.js

```diff
@@ -496,7 +496,7 @@ export const ComputeModalBase = ({
    */
   const getPendingDisk = () => {
     const { persistentDisk: desiredPersistentDisk } = getDesiredEnvironmentConfig()
```

Comment:
How would you feel about using default values in the destructure? This should simplify this and prevent the case where `size` or `diskType` does not exist and gets set to `undefined`
```suggestion
    const { persistentDisk: { size = 0, diskType = pdTypes.standard } = {}  } = getDesiredEnvironmentConfig()
```

Reply:
hmm, didn't think to use a nested destructure. I like this a lot better. Thanks!